### PR TITLE
Proper links on footer

### PR DESCRIPTION
**Pull request by ghophp**
fix footer links..

### CODE REVIEW

File: app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html [hunk 1]
```diff
@@ -189,8 +189,8 @@ <h6 style="color: #666666; font-family: Source Sans Pro, sans-serif; font-weight
...
                                                         <span>&copy; 2015 yetu AG &bull; Version 1.0</span>&nbsp;&nbsp;&nbsp;
-                                                        <a href="http://yetu.me/" target="_blank" style="color: #009900; text-decoration: none;">Privacy Policy</a> -
-                                                        <a href="http://yetu.me/" target="_blank" style="color: #009900; text-decoration: none;">Terms and Conditions</a> -
+                                                        <a href="http://help.yetu.com/privacypolicy" target="_blank" style="color: #009900; text-decoration: none;">Privacy Policy</a> -
+                                                        <a href="http://help.yetu.com/termsandconditions" target="_blank" style="color: #009900; text-decoration: none;">Terms and Conditions</a> -
                                                         <a href="http://yetu.com/imprint/" target="_blank" style="color: #009900; text-decoration: none;">Imprint</a>
```
Comment (ElliDy) [inline app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html]: Links are not working...is that right?

**Comment by ElliDy** [comment]
Should the links not also be added in the real footer of the registration and login pages?

File: app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html (hunk 1 above)
Comment (ghophp) [inline app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html]: Yeah.. Florian asked me to put this links, and he is aware that is not working :D

Comment (ElliDy) [inline app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html]: Ok...then it should be fine ;)

**Comment by ghophp** [comment]
He doesn't mention nothing regarding the links of the registration page, but I believe that we can do it.. let me speak with him and I probably get back to this..

**Comment by jschaul** [comment]
LGTM. But also, add the right links to all pages, not just this one in the email?

**Comment by ghophp** [comment]
Yeah.. I will @jschaul will commit soon.. :+1:

**Pull request merged**